doc/introduction-to-xkb.md

```diff
@@ -0,0 +1,200 @@
+# Introduction to XKB {#xkb-intro}
+
+__XKB__ stands for “X Keyboard Extension”. It may refer to either:
+
+- a [protocol](@ref xkb-the-protocol)
+- a [keyboard layout configuration](@ref xkb-the-config)
+- a [text format](@ref xkb-the-text-format)
+
+## XKB the protocol {#xkb-the-protocol}
+
+A __protocol__ for the [X Windows System], that extends the core protocol.
+
+_xkbcommon’s_ API is somehow derived from this API, but has been
+substantially reworked to expose fewer internal details to clients.
```

Comment:
```suggestion
_xkbcommon’s_ API is somehow derived from this API, but has been
substantially reworked to function as a library instead of a protocol,
and expose fewer internal details to clients.
```

Reply:
Done